data/deprecated.json

```diff
@@ -1253,6 +1253,138 @@
     "old": {"parking": "street"},
     "replace": {"parking": "lane"}
   },
+  {
+    "old": {"parking:lane:both": "yes"},
+    "replace": {"parking:both": "yes"}
+  },
+  {
+    "old": {"parking:lane:both": "no"},
+    "replace": {"parking:both": "no"}
+  },
+  {
+    "old": {"parking:lane:both": "no_parking"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_parking"}
+  },
+  {
+    "old": {"parking:lane:both": "no_standing"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_standing"}
+  },
+  {
+    "old": {"parking:lane:both": "no_stopping"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping"}
+  },
+  {
+    "old": {"parking:lane:both": "fire_lane"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping", "parking:both:restriction:reason": "fire_lane"}
+  },
+  {
+    "old": {"parking:lane:both": "separate"},
+    "replace": {"parking:both": "separate"}
+  },
+  {
+    "old": {"parking:lane:both": "diagonal"},
+    "replace": {"parking:both": "yes", "parking:both:orientation": "diagonal"}
+  },
+  {
+    "old": {"parking:lane:both": "parallel"},
+    "replace": {"parking:both": "yes", "parking:both:orientation": "parallel"}
+  },
+  {
+    "old": {"parking:lane:both": "perpendicular"},
+    "replace": {"parking:both": "yes", "parking:both:orientation": "perpendicular"}
+  },
+  {
+    "old": {"parking:lane:both:parallel": "on_street"},
+    "replace": {"parking:both": "lane"}
+  },
+  {
+    "old": {"parking:lane:left": "yes"},
+    "replace": {"parking:left": "yes"}
+  },
+  {
+    "old": {"parking:lane:left": "no"},
+    "replace": {"parking:left": "no"}
+  },
+  {
+    "old": {"parking:lane:left": "no_parking"},
+    "replace": {"parking:left": "no", "parking:left:restriction": "no_parking"}
+  },
+  {
+    "old": {"parking:lane:left": "no_standing"},
+    "replace": {"parking:left": "no", "parking:left:restriction": "no_standing"}
+  },
+  {
+    "old": {"parking:lane:left": "no_stopping"},
+    "replace": {"parking:left": "no", "parking:left:restriction": "no_stopping"}
+  },
+  {
+    "old": {"parking:lane:left": "fire_lane"},
+    "replace": {"parking:left": "no", "parking:left:restriction": "no_stopping", "parking:left:restriction:reason": "fire_lane"}
+  },
```

Comment:
🟠  110 https://taginfo.openstreetmap.org/tags/parking%3Alane%3Aleft=fire_lane

```suggestion
```

Complex new tagging, so maybe worth it, but better resolved with other means IMO